gate-plugins/src/main/kotlin/com/netflix/spinnaker/gate/plugins/deck/DeckPluginConfiguration.kt

```diff
@@ -32,9 +34,11 @@ open class DeckPluginConfiguration {
   @Bean
   open fun deckPluginCache(
     updateManager: SpinnakerUpdateManager,
-    registry: Registry
-  ): DeckPluginCache =
-      DeckPluginCache(updateManager, PluginBundleExtractor(), registry)
+    registry: Registry,
+    environment: Environment
+  ): DeckPluginCache {
+    return DeckPluginCache(updateManager, PluginBundleExtractor(SpringStrictPluginLoaderStatusProvider(environment)), registry)
```

Comment:
`PluginAutoConfiguration` from kork provides a `SpringStrictPluginLoaderStatusProvider` bean so there's no need to instantiate a new one here.  

For example, I've added this to master:

https://github.com/spinnaker/gate/blob/master/gate-plugins/src/main/kotlin/com/netflix/spinnaker/gate/plugins/deck/DeckPluginConfiguration.kt#L34-L39

Reply:
ah duh! fixing 👍